### PR TITLE
Fix & add FR translations for E3S

### PR DESCRIPTION
**Pull request by Snsei987**

### CODE REVIEW

File: ui/raidboss/data/triggers/e3s.js [hunk 1]
```diff
@@ -115,6 +117,7 @@
         if (data.role == 'healer') {
           return {
             en: 'Tank Busters',
+            fr: 'Tankbursters',
```
Comment (quisquous) [inline ui/raidboss/data/triggers/e3s.js]: Typo?

Comment (Snsei987) [reply ui/raidboss/data/triggers/e3s.js]: I don't know, I've checked other fights and sometimes it's 'Tankburster', sometimes 'Tank burster', not very consistant.

Do you still want me do update it ?

Comment (quisquous) [reply ui/raidboss/data/triggers/e3s.js]: Oh, I meant `Tankbuster` vs `Tankburster`.  I don't see the latter with the extra `r` anywhere else, so was curious if that was intended.

Comment (Snsei987) [reply ui/raidboss/data/triggers/e3s.js]: Oh yeah, mb, gonna fix it

Comment (Snsei987) [reply ui/raidboss/data/triggers/e3s.js]: Should be good now

**Pull request merged**